### ISSUE TITLE
Standard Layout on the Frontend + Editor (Dark): Colors and typography

### ISSUE THREAD

**Issue by merkushin** [Automattic/themes#7124]
According to the provided design, apply spacing, alignment and typography for the Standard (Default) Layout in Course Theme: Dark variation.

## Scope:

- Apply typography based on the design.
- Desktop and mobile.
- Course Theme: Dark variation.
- Standard (Default) Layout.
- Frontend and Editor.

## Out of Scope:
- Don't apply new colorscheme.

## Design

Please check Figma for the design:
8mY8ui4zYK1Ylw2yIFi9Lx-fi-20020045\_37699

**Comment by merkushin** [comment]
For some reason, we can't set styles for Sensei blocks in theme variations.
We're changing our approach. The card is blocked until we solve the issue.

**Comment by Imran92** [comment]
> For some reason, we can't set styles for Sensei blocks in theme variations.
> We're changing our approach. The card is blocked until we solve the issue.

Is it unblocked now?

**Comment by merkushin** [comment]
PR: https://github.com/Automattic/themes/pull/7065
